api/user.js

```diff
@@ -25,7 +25,49 @@ var getUserInfo = function(username, res) {
     var q = new Queue;
     q.series([
         function (lib) {
-            if (username.length <= config.username_length) {
+
+            // Function to create user object out of what's found in DB
+            var storeUserInfoInObj = function(resp) {
+                reporter.log('storeUserInfoInObj() called');
+                if (resp.error) {
+                    response.json({code: 7498,result: 'error', message: resp.error.message}).status(400).pipe(res)
+                    return;
+                } else if (resp.length > 1) {
+                    // Multiple users were returned to the lookup.  Error.
+                    // Error message in the response is intentionally left empty.
+                    reporter.log('Multiple user matched for the user lookup query');
+                    response.json({code: 7498, result: 'error'}).status(400).pipe(res);
```

Comment:
Also, I don't know what these error code are used for, but presumably this error should have a separate one from the other.